src/rt/sections_ldc.d

```diff
@@ -371,18 +377,14 @@ void initSections()
         }
         _dyld_register_func_for_add_image(&scanSections);
     }
-    else version (CRuntime_Microsoft)
+    else version (Windows)
     {
         pushRange(_data_start__, _data_end__);
         if (_bss_start__ != null)
         {
             pushRange(_bss_start__, _bss_end__);
         }
     }
-    else version (Windows)
-    {
-        pushRange(&_data_start__, &_bss_end__);
-    }
```

Comment:
See https://github.com/msysgit/msys/blob/master/winsup/cygwin/winsup.h#L262. I think the previous code was correct - taking the address is necessary, and the old names should match as well.

Reply:
Oh and please don't touch the MSVC part in a functional way, that was working fine when 0.17 was released. ;)